Added Submissions page

DestroyerAlpha opened the pull request:
It is finally here! 🎆 
Submissions page is here!
Please check and report any issues.
#44

DestroyerAlpha committed:
Added Submissions page

DestroyerAlpha committed:
Fixed a redirect bug in display questions

harshraj22 commented:
I think [display_question.php]() has button action set as ```submissions.html``` instead of ```submissions.php```. Can you edit that as well ?

DestroyerAlpha committed:
Fixed url issue and added an update to resources

DestroyerAlpha commented:
> 
> 
> I think display_question.php has button action set as `submissions.html` instead of `submissions.php`. Can you edit that as well ?

fixed

harshraj22 commented:
Also, it seems ques_id is not being passed from display_question.php to submissions.php. Can you check ?

DestroyerAlpha committed:
Changed GET method to POST

DestroyerAlpha commented:
> 
> 
> Also, it seems ques_id is not being passed from display_question.php to submissions.php. Can you check ?

Changed it to POST, now it works! 🤣

JanhaviDadhania commented:
Are you updating "practice questions info" table?
It contains all question Id , total no. of submissions for that question.

and
I think submission page is working even if we don't select any language.

and
It is working even if we directly try to access the page. In that case it won't have any ques_Id to insert into the table... 
![Screenshot from 2019-11-14 14-50-12](https://user-images.githubusercontent.com/46412915/68843617-6eda3a00-06ee-11ea-995a-ed15a595da99.png)

DestroyerAlpha commented:
Is there a table per practice question or for all the practice questions?

DestroyerAlpha commented:
> 
> 
> Is there a table per practice question or for all the practice questions?

@JanhaviDadhania

DestroyerAlpha committed:
Fixed the bugs

DestroyerAlpha commented:
Fixed the bugs mentioned:

- [x] Added a query to update practice_questions_info

- [x] Added a query to add submission in question table

- [x] Added a check on submissions page for direct access!

> 
> 
> Are you updating "practice questions info" table?
> It contains all question Id , total no. of submissions for that question.
> 
> and
> I think submission page is working even if we don't select any language.
> 
> and
> It is working even if we directly try to access the page. In that case it won't have any ques_Id to insert into the table...
> ![Screenshot from 2019-11-14 14-50-12](https://user-images.githubusercontent.com/46412915/68843617-6eda3a00-06ee-11ea-995a-ed15a595da99.png)

DestroyerAlpha committed:
Chaanged GET to POST

DestroyerAlpha committed:
Edited submit.php

harshraj22 reviewed (approved):
LGTM
